docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -68,6 +69,63 @@ filebeat.inputs:
 1. Harvests lines from two files:  `system.log` and `wifi.log`.
 2. Harvests lines from every file in the `apache2` directory, and uses the `fields` configuration option to add a field called `apache` to the output.
 
+## Reading GZIP files
+
+```{applies_to}
+stack: preview 9.2.0
+```
+
+::::{warning}
+This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+::::
```

Comment:
You technically don't need to add this admonition as the `applies_to` badge above has the same warning. 
<img width="423" height="192" alt="Screenshot 2025-08-29 at 9 02 47 AM" src="https://github.com/user-attachments/assets/8f3a416e-f7dd-45e3-81af-c5d0d568f4db" />